index.html

```diff
@@ -124,6 +124,22 @@ <h2>Доп. Информация о расположении и контакты
       <a href="https://htmlacademy.ru">HTML Academy</a>
     </p>
   </footer>
+  <form class="#" action="#" method="post">
+    <button type="button">#</button> <!-- Кнопка закрыть-->
+    <p>
+      <label for="first_and_second_name">Ваше имя:</label>
+      <input id="first_and_second_name" type="text" name="first_and_second_name" value="" placeholder="Имя Фамилия" />
```

Comment:
```suggestion
      <input id="first_and_second_name" type="text" name="first_and_second_name" placeholder="Имя Фамилия">
```
В html5 мы не закрываем одиночные теги. Если у инпута нет значения, то и писать его не надо.

Reply:
Не все вхождения поправил